src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -630,7 +638,11 @@ public WorkspacePanel(final MCreator mcreator) {
 
 		mainp.setOpaque(false);
 
-		modElementsPanel.add("Center", PanelUtils.northAndCenterElement(elementsBreadcrumb, mainp));
+		bcAndDetails.add("Center", PanelUtils.northAndCenterElement(elementsBreadcrumb, detailsbar));
+		detailsbar.add("Center", PanelUtils.gridElements(1, 6, detailsName, detailsID, detailsType, detailsLock, detailsCompile));

```

Comment:
You should directly create your JLabel here if you don't use them anywhere else.

Reply:
what do you mean?

Reply:
Convert class fields for these labels to local variables somewhere near this statement.

Reply:
Also, since you defined `detailsbar` as a panel that uses `GridLayout`, the usage of `gridElements()` becomes obsolete. You should replace this line with adding labels one by one (you can even add them without declaring them as local variables):
```
		detailsbar.add(detailsName);
		detailsbar.add(detailsID);
		detailsbar.add(detailsType);
		detailsbar.add(detailsLock);
		detailsbar.add(detailsCompile);

```

Reply:
> Convert class fields for these labels to local variables somewhere near this statement.

No, what I meant is to create directly those labels in the method adding the labels to the panel. Something like `panel.add(PanelUtils.join(L10N.label(blabla), L10N.label(blablabla)));`

Reply:
I tried to use GridElement() but it got weird layout after

Reply:
I tried did your thing goldorion